### PR TITLE
Update EMsg and some enums

### PR DESCRIPTION
**Pull request by xPaw**

### CODE REVIEW

**Comment by yaakov-h** [comment]
What's the source on these?

If you got the EMsgs from Steam, please also update emsg_list_detailed.txt.

**Comment by xPaw** [comment]
nethook for emsg, and steamdb tracker for enums

File: Resources/Misc/emsg_list_detailed.txt [hunk 1]
```diff
@@ -420,11 +424,11 @@
 	k_EMsgClientFSGetIsFollowingResponse = 7518, // flags: 6, server type: 47
 	k_EMsgClientFSEnumerateFollowingList = 7519, // flags: 6, server type: 47
 	k_EMsgClientFSEnumerateFollowingListResponse = 7520, // flags: 6, server type: 47
-	k_EMsgClientFSOfflineMessageNotification = 7523, // flags: 2, server type: -1
-	k_EMsgClientFSRequestOfflineMessageCount = 7524, // flags: 6, server type: 47
-	k_EMsgClientFSGetFriendMessageHistory = 7525, // flags: 6, server type: 47
-	k_EMsgClientFSGetFriendMessageHistoryResponse = 7526, // flags: 6, server type: 47
-	k_EMsgClientFSGetFriendMessageHistoryForOfflineMessages = 7527, // flags: 6, server type: 47
+	k_EMsgClientChatOfflineMessageNotification = 7523, // flags: 2, server type: -1
```
Comment (yaakov-h) [inline Resources/Misc/emsg_list_detailed.txt]: This should have a corresponding EMsg update... we usually leave the old name in as an obsoleted alias for one release.

**Comment by xPaw** [comment]
@yaakov-h Done, I think?

File: Resources/SteamLanguage/emsg.steamd [hunk 2]
```diff
@@ -1691,11 +1695,16 @@ enum EMsg
...
-	ClientFSOfflineMessageNotification = 7523;
-	ClientFSRequestOfflineMessageCount = 7524;
-	ClientFSGetFriendMessageHistory = 7525;
-	ClientFSGetFriendMessageHistoryResponse = 7526;
-	ClientFSGetFriendMessageHistoryForOfflineMessages = 7527;
+	ClientFSOfflineMessageNotification = 7523; removed "Renamed to ClientChatOfflineMessageNotification"
+	ClientFSRequestOfflineMessageCount = 7524; removed "Renamed to ClientChatRequestOfflineMessageCount"
+	ClientFSGetFriendMessageHistory = 7525; removed "Renamed to ClientChatGetFriendMessageHistory"
+	ClientFSGetFriendMessageHistoryResponse = 7526; removed "Renamed to ClientChatGetFriendMessageHistoryResponse"
+	ClientFSGetFriendMessageHistoryForOfflineMessages = 7527; removed "Renamed to ClientChatGetFriendMessageHistoryForOfflineMessages"
```
Comment (yaakov-h) [inline Resources/SteamLanguage/emsg.steamd]: Use `obsolete` instead of `removed` to continue generating the C# code for this enum value.

**Comment by xPaw** [comment]
Dooooooooone

**Pull request merged**